src/AAVE/Modal/SupplyModal.jsx

```diff
@@ -356,7 +356,7 @@ return (
                   const amount = Big(state.amount)
                     .mul(Big(10).pow(decimals))
                     .toFixed(0);
-                  if (symbol === "WETH") {
+                  if (symbol === "ETH") {
```

Comment:
```suggestion
                  if (symbol === "ETH" || symbol === "WETH") {
```

Should we process ETH and WETH the same way? Does this only apply to ETH?